### ISSUE TITLE
RFI First Page

### ISSUE THREAD

**Issue by naveen1601** [OrderMail/blacklabel#15]
###Acceptance Criteria:

Create a page for RFI
1.Create Three Fields-
    1.1 To
    1.2 Subject
    1.3 Body
2. Two Buttons-
    2.1 Send Button 
   2.2 Draft

Send/Draft Functionality- Save RFI page data into database.

**Comment by vikasagarwal** [comment]
@naveen1601 is it in Dev? If so, please change the label appropriately. Also, update with the current status and your questions.

**Comment by amit-k86** [comment]
@naveen1601 @vikasagarwal @priyankshukla - I have added following mock ups. Have changed fields. Will discuss it. Request you to review in the meantime.

![mock up](https://cloud.githubusercontent.com/assets/9941976/6319981/162880ac-baf8-11e4-8130-8d4704a1b232.jpg)

![inspiration](https://cloud.githubusercontent.com/assets/9941976/6319987/2e7faf9a-baf8-11e4-8db6-75ac4ab81fcc.jpg)

![3rd rfi](https://cloud.githubusercontent.com/assets/9941976/6319986/2db60f82-baf8-11e4-9839-873bb92517ae.jpg)

![4rd page](https://cloud.githubusercontent.com/assets/9941976/6319988/2e80d898-baf8-11e4-94e2-dfee733b791a.jpg)

**Comment by naveen1601** [comment]
We are moving forward with @amit-k86 's design.

###Acceptance Criteria

![](https://cloud.githubusercontent.com/assets/9941976/6319981/162880ac-baf8-11e4-8130-8d4704a1b232.jpg)

This page will contains following details - 
- [ ] Choose A Business - It is a dropdown and will contains all the 
- [ ] Enter email address
- [ ] Mailing Address - It is a address of the company, auto fill from company object
- [ ] Ship To 
- [ ] RFI Date - Current date (default), it can be change by user.
- [ ] RFI Due Date - When this RFI going to expire
- [ ] Shipping Address -  Auto fill from company object
- [ ] Item Details - It is in a tabular form- item number, name, etc. 
- [ ] Message -  Textarea for discription and message.
- [ ] Attachments - Attachments like - pdf, jpg, doc, etc can be attach.
- [ ] Approved By - Text box, name of a person
- [ ] Sign -  Digital signature
- [ ] Date - Current Date (default)

**Comment by amit-k86** [comment]
@naveen1601 @vikasagarwal  another thought- gen RFI send to multiple business's.. So option to be provided to user to choose another business in case he want to send the same to multiple users. Can be done later but just thought was worth mentioning so can be discussed later.

**Comment by vikasagarwal** [comment]
Indeed, that's a good idea. RFI and RFQ's make sense if are sent to
multiple parties without additional efforts. Let's design that as well and
make a decision about when we want to take on that.

On Wed, Mar 18, 2015 at 6:56 AM, amit-k86 <notifications@github.com> wrote:

> @naveen1601 <https://github.com/naveen1601> @vikasagarwal
> <https://github.com/vikasagarwal> another thought- gen RFI send to
> multiple business's.. So option to be provided to user to choose another
> business in case he want to send the same to multiple users. Can be done
> later but just thought was worth mentioning so can be discussed later.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OrderMail/blacklabel/issues/15#issuecomment-82665599>.
>



-- 
Best Cricket T20 Coverage on:
http://cricket-t20.com

**Comment by priyankshukla** [comment]
It's been long time, no update..
Kindly update the latest status of this issue.
